### PR TITLE
CORE-19485: Implement RBAC password length restriction

### PR DESCRIPTION
**Pull request by vkolomeyko**
Unit test been updated and also I did some manual testing locally with Combined Worker.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -42,7 +42,7 @@ commonsLangVersion = 3.12.0
 commonsTextVersion = 1.10.0
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.2.0.xx-SNAPSHOT to pick up maven local published copy
-cordaApiVersion=5.2.0.33-beta+
+cordaApiVersion=5.2.0.33-alpha-1706694678581
```
Comment (vkolomeyko) [inline gradle.properties]: - [x] Undo this change once `corda-api` PR is merged: https://github.com/corda/corda-api/pull/1475

**Review by josephzunigadaly — APPROVED** [review]
